marketplace-core/pom.xml

```diff
@@ -97,6 +97,10 @@
       <groupId>org.webjars.bower</groupId>
       <artifactId>angular-ui-bootstrap-bower</artifactId>
     </dependency>
+    <dependency>
```

Comment:
big-data-plugin uses version of 2015: https://github.com/pentaho/big-data-plugin/blob/56f061ec426ad7e31e5985d549edd8bad2988d69/pom.xml